### PR TITLE
Release 0.11

### PR DESCRIPTION
**Pull request by rafalp**

### CODE REVIEW

File: docs/resolvers.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ class FormResolver:
...
 
-> `context` is just one of many attributes that can be found on `GraphQLResolveInfo`, but it is by far the most commonly used one. Other attributes enable developers to introspect the query that is currently executed and implement new utilities and abstractions, but documenting that is out of Ariadne's scope. If you are interested, you can find the list of all attributes [here](https://github.com/graphql-python/graphql-core-next/blob/v1.0.5/graphql/type/definition.py#L487).
+> `context` is just one of many attributes that can be found on `GraphQLResolveInfo`, but it is by far the most commonly used one. Other attributes enable developers to introspect the query that is currently executed and implement new utilities and abstractions, but documenting that is out of Ariadne's scope. If you are interested, you can find the list of all attributes [here](https://github.com/graphql-python/graphql-core/blob/v1.0.5/graphql/type/definition.py#L487).
```
Comment (patrys) [inline docs/resolvers.md:22]:
```suggestion
> `context` is just one of many attributes that can be found on `GraphQLResolveInfo`, but it is by far the most commonly used one. Other attributes enable developers to introspect the query that is currently executed and implement new utilities and abstractions, but documenting that is out of Ariadne's scope. If you are interested, you can find the list of all attributes [here](https://github.com/graphql-python/graphql-core/blob/v3.0.3/src/graphql/type/definition.py#L533-L553).
```

File: docs/scalars.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ Expected type Datetime!, found "invalid string"; time data 'invalid string' does
 
 > Because the error message returned by the GraphQL server includes the original exception message from your Python code, it may contain details specific to your system or implementation that you may not want to make known to the API consumers. You may decide to catch the original exception with `except (ValueError, TypeError)` and then raise your own `ValueError` with a custom message (or no message at all) to prevent this from happening.
 
-If a value is specified as part of query content, its `ast` node is instead passed to `parse_datetime_literal` to give the scalar a chance to introspect the type of the node (implementations for those be found [here](https://github.com/graphql-python/graphql-core-next/blob/master/graphql/language/ast.py#L261)).
+If a value is specified as part of query content, its `ast` node is instead passed to `parse_datetime_literal` to give the scalar a chance to introspect the type of the node (implementations for those be found [here](https://github.com/graphql-python/graphql-core/blob/v1.0.5/graphql/language/ast.py#L265)).
```
Comment (patrys) [inline docs/scalars.md:140]:
```suggestion
If a value is specified as part of query content, its `ast` node is instead passed to `parse_datetime_literal` to give the scalar a chance to introspect the type of the node (implementations for those be found [here](https://github.com/graphql-python/graphql-core/blob/v3.0.3/src/graphql/language/ast.py#L344)).
```

File: docs/types-reference.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ Should return a Python `dict`.
 
 #### `error`
 
-Instance of [`graphql.error.GraphQLError`](https://github.com/graphql-python/graphql-core-next/blob/v1.0.5/graphql/error/graphql_error.py#L15) to be formatted.
+Instance of [`graphql.error.GraphQLError`](https://github.com/graphql-python/graphql-core/blob/v1.0.5/graphql/error/graphql_error.py#L15) to be formatted.
```
Comment (patrys) [inline docs/types-reference.md:49]: Still v1.0.5 in this file

**Comment by rafalp** [comment]
@patrys updated all graphql-core source links to point to v3.0.3

**Pull request merged**